aspnetcore/performance/caching/memory/sample/RPcache/Pages/About.cshtml.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Caching.Memory;
+using RPcache.Services;
+using System;
+
+namespace RPcache.Pages
+{
+    #region snippet
+    public class AboutModel : PageModel
+    {
+        private MemoryCache _cache;
+        public static readonly string MyKey = "_MyKey";
+
+        public AboutModel(MyMemoryCache memoryCache)
+        {
+            _cache = memoryCache.Cache;
+        }
+
+        [TempData]
+        public string DateTime_Now { get; set; }
+
+        #region snippet2
+        public IActionResult OnGet()
+        {
+            if (!_cache.TryGetValue(MyKey, out string cacheEntry))
+            {
+                // Key not in cache, so get data.
+                cacheEntry = DateTime.Now.TimeOfDay.ToString();
+
+                var cacheEntryOptions = new MemoryCacheEntryOptions()
+                    // Set cache entry size via extension method.
+                    .SetSize(1)
```

Comment:
Comment on the units you've decided to use to measure the size, and what some alternative examples may be.